mne/io/meas_info.py

```diff
@@ -1190,6 +1196,21 @@ def create_info(ch_names, sfreq, ch_types=None):
                          unit=kind[2], coord_frame=FIFF.FIFFV_COORD_UNKNOWN,
                          ch_name=name, scanno=ci + 1, logno=ci + 1)
         info['chs'].append(chan_info)
+    if montage is not None:
+        from ..channels.montage import (Montage, DigMontage, _set_montage,
+                                        read_montage)
+        if not isinstance(montage, list):
+            montage = [montage]
+        for m in montage:
+            if isinstance(m, (Montage, DigMontage)):
+                _set_montage(info, m)
+            elif isinstance(m, string_types):
+                m = read_montage(m)
+                _set_montage(info, m)
```

Comment:
let's please try to avoid single letter variable names. Maybe `this_montage` or `montage_` .